subsys/cassandra/src/main/java/org/commonjava/indy/subsys/cassandra/CassandraClient.java

```diff
@@ -71,8 +71,8 @@ private void init()
         host = config.getCassandraHost();
         port = config.getCassandraPort();
         SocketOptions socketOptions = new SocketOptions();
-        socketOptions.setConnectTimeoutMillis( 30000 );
-        socketOptions.setReadTimeoutMillis( 30000 );
+        socketOptions.setConnectTimeoutMillis( 120000 );
+        socketOptions.setReadTimeoutMillis( 120000 );
```

Comment:
For the timeout issue, we may need to introduce the retry policy, https://docs.datastax.com/en/developer/java-driver/3.6/manual/retries/
Let's try that later if needed.